sherpa-onnx/csrc/online-model-config.cc

```diff
@@ -56,7 +56,7 @@ bool OnlineModelConfig::Validate() const {
     return false;
   }
 
-  if (!FileExists(tokens)) {
+  if (tokens_buf_str.empty() && !FileExists(tokens)) {
```

Comment:
Please validate that either tokens is empty or tokens_buf_str is empty.

The cannot be non-empty or empty at the same time.